.github/workflows/func_spec.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [windows-2019, windows-2016]
+        os: [windows-2019, windows-2022]
         # Due to https://github.com/actions/runner/issues/849, we have to use quotes for '3.0'
         ruby: [2.7, '3.0']
```

Comment:
We are not yet testing this on Ruby 3.1, correct?

Reply:
Correct, working towards that, need to get these changes in first